tests/test_autoingest.py

```diff
@@ -207,13 +209,19 @@ async def testBadIngest(self):
 
         ingester = FileIngester(config["ingester"])
 
-        await ingester.ingest([self.destFile])
+        print(f"1 self.destFile = {self.destFile}")
```

Comment:
Looks like a forgotten remnant of some debugging session. Please remove here and below.

Reply:
fixed